support/docker/opt/config/server.cfg

```diff
@@ -0,0 +1,62 @@
+radar.playerunitcolor = 0
+radar.selectedunitcolor = 13
+radar.alliedunitcolor = 11
+radar.playeroutpostcolor = 1
+radar.alliedoutpostcolor = 11
+radar.enemyoutpostcolor = 4
+radar.unitsize = 0
+player.name = "NetPanzer docker server"
+server.port = 3031
+server.bindaddress = ""
+server.motd = "Welcome to NetPanzer"
+server.logging = false
+server.public = true
+server.masterservers = "netpanzer.io"
+server.name = "NetPanzer docker server"
+interface.show_health = true
+interface.show_flags = true
+interface.show_names = true
+interface.attacknotificationime = 5
+interface.vehicleselectioncolor = 1
+interface.unitselectionmode = 1
+interface.unitinfodrawlayer = 0
+interface.scrollrate = 1000
+interface.rankposition_x = 100
+interface.rankposition_y = 100
+interface.viewdrawbackgroundmode = 0
+game.enable_bases = true
+game.base_capture_mode = 1
+game.base_limit = 0
+game.autokicktime = 20
+game.allowmultiip = true
+game.unit_profiles = "Manta, Panther1, Titan, Stinger, Bobcat, Bear, Archer, Wolf, Drake, Spanzer"
+game.unit_spawnlist = "1, 1, 1, 1, 1, 1, 1, 1, 1, 1"
+game.adminpass = ""
+game.gamepass = ""
+game.changeflagtime = 1
+game.gametype = 0
+game.maxplayers = 8
+game.maxunits = 288
+game.timelimit = 30
+game.fraglimit = 300
+game.powerups = true
+game.occupationpercentage = 75
+game.allowallies = true
+game.cloudcoverage = 0
+game.respawntype = 0
+game.windspeed = 30
+game.lowscorelimit = -25
+game.map = "Two clans"
+game.mapcycle = "Two clans, Bullet Hole"
+sound.enable = true
+sound.music = true
+sound.musicvol = 100
+sound.effects = true
+sound.effectsvol = 100
+video.width = 800
+video.height = 600
+video.fullscreen = false
+video.hardwaresurface = false
+video.doublebuffer = false
+video.shadows = true
+video.blendsmoke = true
```

Comment:
I think without setting `server.interactive_console` to `false` it will just spin on 100% cpu on one core.

Reply:
fixed